### PR TITLE
RandomApiMigrationFixer - better docs and better "random_int" support

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Review by julienfalque — APPROVED** [review]

**Review by kubawerlos — APPROVED** [review]

File: src/Fixer/Alias/RandomApiMigrationFixer.php [hunk 1]
```diff
@@ -162,7 +165,7 @@ protected function createConfigurationDefinition()
                 }])
                 ->setDefault([
                     'getrandmax' => 'mt_getrandmax',
-                    'rand' => 'mt_rand',
+                    'rand' => 'mt_rand', // TODO change to `random_int` as default on 4.0
```
Comment (keradus) [inline src/Fixer/Alias/RandomApiMigrationFixer.php:168]:
```suggestion
                    'rand' => 'mt_rand', // @TODO change to `random_int` as default on 4.0
```

so I can find it later

File: tests/Fixer/Alias/RandomApiMigrationFixerTest.php [hunk 2]
```diff
@@ -221,4 +221,14 @@ public function provideFix73Cases()
             ],
         ];
     }
+
+    public function testFixToRandomInt()
```
Comment (keradus) [inline tests/Fixer/Alias/RandomApiMigrationFixerTest.php:225]: we already have tests for `random_int`:
- without params: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.19/tests/Fixer/Alias/RandomApiMigrationFixerTest.php#L96-L100
- with params: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.19/tests/Fixer/Alias/RandomApiMigrationFixerTest.php#L163-L167

**Comment by keradus** [comment]
> RandomApiMigrationFixer - add option to migrate to "random_int"

it's *not* adding any option - one can already migrate to `random_int`. Even `PHP70MigrationRisky` set is doing that

**Review by keradus — CHANGES_REQUESTED** [review]
- please drop duplicated tests
- this is not enhancement PR, I see no new possibilities coming with this PR, the main value I see is documenting not-fully-documented random_int option

File: tests/Fixer/Alias/RandomApiMigrationFixerTest.php (hunk 2 above)
Comment (keradus) [reply tests/Fixer/Alias/RandomApiMigrationFixerTest.php:225]: @SpacePossum can you take a look pls?

**Comment by SpacePossum** [comment]
I've removed the tests

**Comment by SpacePossum** [comment]
I was wrong thinking adding support here indeed. It is because this is also supported currently:
```
            [
                '<?php foo();',
                '<?php rand();',
                ['replacements' => ['rand' => 'foo']],
            ],
```
making it a more generic function renaming rule than I expected. I was expecting known PHP random methods as input _and_ as output.
Anyway, besides the docs this PR also adds proper argument counts for using `random_int` as input FWIW

**Comment by SpacePossum** [comment]
sca fail seems unrelated

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**